apps/web/package.json

```diff
@@ -103,6 +102,7 @@
     "@types/react-dom": "18.3.0",
     "autoprefixer": "^10.4.19",
     "drizzle-kit": "catalog:",
+    "import-in-the-middle": "^1.14.2",
```

Comment:
Not used no?

Reply:
yes it's to remove the annoying message in dev mode